### PR TITLE
Post NPQ Separation Repointing

### PR DESCRIPTION
**Pull request by AntPage1**
This PR contains all of the re-pointed marts that need to go live in order to start pushing up to date NPQ data to our dashboards. TAD Marts are being repointed to archived tables for NPQ Enrolments and Participant Outcomes so that these do not change until TAD agree they are ready to receive the new table schemas.

### CODE REVIEW

File: definitions/marts/npq_separation_prep/ls_declarations_provider_names_v2.sqlx
Comment (consult1815) [inline definitions/marts/npq_separation_prep/ls_declarations_provider_names_v2.sqlx]: You don't need DISTINCT with a group by

File: definitions/marts/npq_separation_prep/npq_enrolments_v2.sqlx
Comment (consult1815) [inline definitions/marts/npq_separation_prep/npq_enrolments_v2.sqlx]: I have concerns about linking this to the user only and not finding a way to connect it with the application. What would happen if this individual was to re-apply and mis-enter their TRN? Is there any way to capture that, could that happen?

File: definitions/marts/npq_separation_prep/tad_npq_participant_declarations.sqlx
Comment (consult1815) [inline definitions/marts/npq_separation_prep/tad_npq_participant_declarations.sqlx:23]: I don't think this description is sufficiently clear for TAD. I don't think GUID in general is sufficiently descriptive to explain how NPQ are using it (in any mart let alone a TAD mart).

File: definitions/marts/bigquery_marts/npq/npq_enrolments.sqlx
Comment (consult1815) [inline definitions/marts/bigquery_marts/npq/npq_enrolments.sqlx:4]: Is this identical to the version I approved before I went on leave? If so I'll approve it, otherwise can you point me to any changes?

File: definitions/marts/looker_studio_marts/npq/ls_api_errors_npq_outcomes.sqlx
Comment (consult1815) [inline definitions/marts/looker_studio_marts/npq/ls_api_errors_npq_outcomes.sqlx:13]: Same question, is this identical to the version I approved?

File: definitions/marts/looker_studio_marts/ls_declarations_provider_names.sqlx
Comment (consult1815) [inline definitions/marts/looker_studio_marts/ls_declarations_provider_names.sqlx:10]: Same approval question.

File: definitions/marts/looker_studio_marts/npq/ls_npqeyl_enrolments.sqlx
Comment (consult1815) [inline definitions/marts/looker_studio_marts/npq/ls_npqeyl_enrolments.sqlx:4]: Same question.

File: definitions/marts/looker_studio_marts/ls_participant_profile_states.sqlx
Comment (consult1815) [inline definitions/marts/looker_studio_marts/ls_participant_profile_states.sqlx:12]: Same Question

File: definitions/marts/looker_studio_marts/npq/ls_npq_acceptance_reversal.sqlx
Comment (consult1815) [inline definitions/marts/looker_studio_marts/npq/ls_npq_acceptance_reversal.sqlx:22]: Could this sum still contain duplicates if there were overlaps in the UNION?

**Comment by consult1815** [comment]
Just a few points of clarification also haven't looked over the tables copied and pasted from V2, let me know if those changed at all when you moved them over and I'll check the changes.

**Review by consult1815 — CHANGES_REQUESTED** [review]
Just a few points of clarification also haven't looked over the tables copied and pasted from V2, let me know if those changed at all when you moved them over and I'll check the changes.

File: definitions/marts/bigquery_marts/npq/npq_enrolments.sqlx
Comment (AntPage1) [reply definitions/marts/bigquery_marts/npq/npq_enrolments.sqlx:4]: It is the same as the V2 file you approved previously.

File: definitions/marts/looker_studio_marts/ls_declarations_provider_names.sqlx
Comment (AntPage1) [reply definitions/marts/looker_studio_marts/ls_declarations_provider_names.sqlx:10]: It is the same as the V2 file you approved previously.

File: definitions/marts/looker_studio_marts/npq/ls_npqeyl_enrolments.sqlx
Comment (AntPage1) [reply definitions/marts/looker_studio_marts/npq/ls_npqeyl_enrolments.sqlx:4]: It is the same as the V2 file you approved previously.

File: definitions/marts/looker_studio_marts/ls_participant_profile_states.sqlx
Comment (AntPage1) [reply definitions/marts/looker_studio_marts/ls_participant_profile_states.sqlx:12]: It is the same as the V2 file you approved previously.

File: definitions/marts/looker_studio_marts/npq/ls_npq_acceptance_reversal.sqlx
Comment (AntPage1) [reply definitions/marts/looker_studio_marts/npq/ls_npq_acceptance_reversal.sqlx:22]: By doing the grouping on the result of the UNION it returns a single row per application ultimately. This was implemented after testing as there were duplicates cause by applications appearing in both halves of the union.

File: definitions/marts/npq_separation_prep/npq_enrolments_v2.sqlx
Comment (AntPage1) [reply definitions/marts/npq_separation_prep/npq_enrolments_v2.sqlx]: I have reworked this section on npq_enrolments. Please review and let me know your thoughts on the proposed method.

**Review by consult1815 — APPROVED** [review]
Looks good

**Pull request merged**
